plugins/inlinetoolbar/skins/default.css

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or https://ckeditor.com/license
+*/
+
+.cke_balloon.cke_inlinetoolbar
+{
+	position: absolute;
+	z-index: 10000;
+	outline: none;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle
+{
+	position: absolute;
+	border-style: solid;
+	display: block;
+	width: 0;
+	height: 0;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer
+{
+	z-index: 0;
+}
+
+/* align: [ hcenter ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_align_hcenter
+{
+	left: 50%;
+	margin-left: -20px;
+}
+
+.cke_balloon.cke_inlinetoolbar
+{
+	background: #f8f8f8;
+	border: 1px solid #d1d1d1;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_content
+{
+	min-height: 10px;
+	padding: 3px;
+}
+
+/* side: [ bottom, top ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_bottom,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_top
+{
+	border-color: #d1d1d1 transparent;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_bottom,
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_top
+{
+	border-color: #f8f8f8 transparent;
+	left: -10px;
+}
+
+/* side: [ bottom ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_bottom
+{
+	border-width: 10px 10px 0;
+	bottom: -10px;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_bottom
+{
+	border-width: 10px 10px 0;
+	top: -11px;
+}
+
+/* side: [ top ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_top
+{
+	border-width: 0 10px 10px;
+	top: -10px;
+}
+
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_inner.cke_balloon_triangle_top
+{
+	border-width: 0 10px 10px;
+}
+
+/* align: [ hcenter ] */
+.cke_balloon.cke_inlinetoolbar .cke_balloon_triangle_outer.cke_balloon_triangle_align_hcenter
```

Comment:
This is the same selector as in line 28 (https://github.com/ckeditor/ckeditor-dev/pull/1136/files#diff-a222e1835c2e1c3b856fdaf98d45e2faR28), it should be merged into one rule.